sdk/communication/communication-chat/src/generated/src/models/index.ts

```diff
@@ -401,36 +445,21 @@ export interface ChatThreadSendTypingNotificationOptionalParams
 
 /** Optional parameters. */
 export interface ChatThreadListChatReadReceiptsNextOptionalParams
-  extends coreClient.OperationOptions {
-  /** The maximum number of chat message read receipts to be returned per page. */
-  maxPageSize?: number;
-  /** Skips chat message read receipts up to a specified position in response. */
-  skip?: number;
-}
```

Comment:
Could you please check if the params in OperationOptions?

Reply:
Yeah, this is being removed in beta release already. cc John